frontend/src/metabase-lib/v1/metadata/utils/models.ts

```diff
@@ -86,21 +86,12 @@ function isSupportedTemplateTagForModel(tag: TemplateTag) {
   return ["card", "snippet"].includes(tag.type);
 }
 
-function checkDatabaseSupportsModels(database?: Database | null) {
-  return database && database.hasFeature("nested-queries");
-}
-
 export function checkDatabaseCanPersistDatasets(database?: Database | null) {
   return database && database.supportsPersistence() && database.isPersisted();
 }
 
 export function checkCanBeModel(question: Question) {
-  if (!checkDatabaseSupportsModels(question.database())) {
```

Comment:
Change 2 - this code disallowed converting a question into a model for nodata users because they didn't have access to `database`